superset/daos/base.py

```diff
@@ -127,57 +128,72 @@ def find_one_or_none(cls, **filter_by: Any) -> Optional[T]:
         return query.filter_by(**filter_by).one_or_none()
 
     @classmethod
-    def create(cls, properties: dict[str, Any], commit: bool = True) -> T:
-        """
-        Generic for creating models
-        :raises: DAOCreateFailedError
+    def create(
+        cls,
+        item: T | None = None,
```

Comment:
A accommodate the `save` functionality I extended the `create` method to accept a predefined object.

Reply:
Would be possible to set `item` as `T | dict[str, Any] | None` and remove the `attributes` parameter?

Reply:
@michael-s-molina that's an interesting idea. I was likely just trying to replicate the same behavior as the `update` method which needs to have either/or both the item and attributes—well technically you're just updating an existing item, but it seems cleaner (from a DRY perspective) to have the base `update` method perform said logic.

Reply:
Technically I can call `create` with both `item` and `attributes` set as `None`. It feels really weird for me.

Reply:
@michael-s-molina sorry for the delay in getting back to you on this. You mention you could do something of the form,

```python
chart = ChartDAO.create()
```

which would try to create a new chart and persist it to the database. This operation would succeed if the model attributes can be nullable. I realize this isn't a typical workflow, but I'm not entirely sure it's wrong per se.